src/ui/js/Game.js

```diff
@@ -3019,10 +3020,10 @@ Game.createDieContainerDiv = function(
  * Return a game-mat-style display of all dice belonging to the
  * requested player
  *
- * @param  string  player        Whose dice to display ('player' or 'opponent')
- * @param  boolean player_active Is the game awaiting action from the player
- *                               who is loading the page?
- * @return object                jQuery containing the div of dice
+ * @param  {string}  player        Whose dice to display ('player' / 'opponent')
```

Comment:
I'm running up against the 80 character limit here. Would you prefer 'or' with less aligned spacing or the '|' symbol?